pkgs/development/libraries/audio/zita-resampler/default.nix

```diff
@@ -13,7 +14,7 @@ stdenv.mkDerivation rec {
     "SUFFIX="
   ];
 
-  patchPhase = ''
+  postPatch = ''
     cd source
     sed -e "s@ldconfig@@" -i Makefile
   '';
```

Comment:
Can you please rebase with `master`? This is not its current contents: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/audio/zita-resampler/default.nix#L16-L23